assets/css/main.css

```diff
@@ -60,4 +60,109 @@ b {
     & h3 {
     min-width: 10rem;
   }
-}
\ No newline at end of file
+}
+
+@property --load-position {
+  syntax: "<angle>";
+  initial-value: -270deg;
+  inherits: true;
+}
+
+@property --load-offset {
+  syntax: "<angle>";
+  initial-value: 270deg;
+  inherits: true;
+}
+
+@property --trail-offset {
+  syntax: "<angle>";
+  initial-value: 0deg;
+  inherits: true;
+}
+
+.loader-outside {
+  color-scheme: light dark;
+
+  --bg: light-dark(white, #333);
+  --loader-color: light-dark(dodgerblue, deepskyblue);
```

Comment:
We don't actually have any background themeing yet, so this is giving me a dark inner circle 😅. 

We can probably leave it as white background for now. Thanks!